hq/public/stylesheets/main.css

```diff
@@ -200,8 +201,16 @@ nav {
 
 /* security group pages */
 
+.sg-filter__label {
+  margin-left: 20px;
+}
+
 .sg-header__name {
-  margin-right: auto;
+  flex-grow: 2;
+}
+
+.sg-header span.sg-header__badge {
+  margin-left: 10px;
```

Comment:
Boo specificity! 👎 
Unfortunately, this is necessary to win a fight with materializecss because multiple badges should not collide (IMHO)

Reply:
😄